make/scripts/fixpath.sh

```diff
@@ -146,15 +146,11 @@ function import_path() {
   fi
 
   if [[ "$path" != "" ]]; then
-    # Store current unix path
-    unixpath="$path"
```

Comment:
I think the distinction between "unixpath" and "winpath" was helpful, with "path" being the input/output variable of this conversion process. Was there some specific reason that you've removed "unixpath"?